operator/src/main/java/oracle/kubernetes/weblogic/domain/model/Domain.java

```diff
@@ -501,10 +502,144 @@ public boolean isHttpAccessLogInLogHome() {
     return spec.getHttpAccessLogInLogHome();
   }
 
+  /**
+   * Returns if the domain is using online update.
+   * return true if using onlne update
+   */
+
+  public boolean isUseOnlineUpdate() {
+    return Optional.ofNullable(spec)
+        .map(DomainSpec::getConfiguration)
+        .map(Configuration::getModel)
+        .map(Model::getOnlineUpdate)
+        .map(OnlineUpdate::getEnabled)
+        .orElse(false);
+  }
+
+  /**
+   * Returns WDT activate timeout.
+   * @return WDT activate timeout
+   */
+  public Long getWDTActivateTimeout() {
+    return Optional.ofNullable(spec)
+        .map(DomainSpec::getConfiguration)
+        .map(Configuration::getModel)
+        .map(Model::getOnlineUpdate)
+        .map(OnlineUpdate::getActivateTimeoutMilliSeconds)
+        .orElse(null);
+  }
+
+  /**
+   * Returns WDT connect timeout.
+   * @return WDT connect timeout
+   */
+  public Long getWDTConnectTimeout() {
+    return Optional.ofNullable(spec)
+        .map(DomainSpec::getConfiguration)
+        .map(Configuration::getModel)
+        .map(Model::getOnlineUpdate)
+        .map(OnlineUpdate::getConnectTimeoutMilliSeconds)
+        .orElse(null);
+  }
+
+  /**
+   * Returns WDT deploy timeout.
+   * @return WDT deploy timeout
+   */
+  public Long getWDTDeployTimeout() {
+    return Optional.ofNullable(spec)
+        .map(DomainSpec::getConfiguration)
+        .map(Configuration::getModel)
+        .map(Model::getOnlineUpdate)
+        .map(OnlineUpdate::getDeployTimeoutMilliSeconds)
+        .orElse(null);
+  }
+
+  /**
+   * Returns WDT undeploy timeout.
+   * @return WDT undeploy timeout
+   */
+  public Long getWDTUnDeployTimeout() {
+    return Optional.ofNullable(spec)
+        .map(DomainSpec::getConfiguration)
+        .map(Configuration::getModel)
+        .map(Model::getOnlineUpdate)
+        .map(OnlineUpdate::getUndeployTimeoutMilliSeconds)
+        .orElse(null);
+  }
+
+  /**
+   * Returns WDT redeploy timeout.
+   * @return WDT redeploy timeout
+   */
+  public Long getWDTReDeployTimeout() {
```

Comment:
Minor nit:  All of these timeouts are in Millis - so end all of their method names with Millis to make that clear...

Reply:
done